components/blog.js

```diff
@@ -60,19 +60,19 @@ export default withView(props => {
                   />
                   <small className="ml-2">{author}</small>
                   <small className="before:content-['/'] before:p-0 before:m-2">
-                    {date}
+                    <time dateTime={date}>{date}</time>
```

Comment:
https://github.com/facebook/react/issues/11492

`dateTime` here can be regonized by browsers even it's not transfered to lowercase `datetime`. That's ok.